api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -36,9 +37,13 @@
 
   @AuthorityRequired({Authority.DEVELOPER})
   @Override
-  public ResponseEntity<EmptyResponse> createAuthDomain(String groupName) {
-    fireCloudService.createGroup(groupName);
-    return ResponseEntity.ok(new EmptyResponse());
+  public ResponseEntity<AuthDomainCreatedResponse> createAuthDomain(String authDomainName) {
```

Comment:
Step 45 of the [New Application Environment creation doc](https://docs.google.com/document/d/1_vZyDWrVYkJWivBDpkMLrf16DHLg2p7qK8wDFdk-784/edit#) is running the Create Auth Domain command.

Step 46 involves determining the Auth Domain's Terra Group Email via a multi-step process.  Terra already returns this information to us during creation!  Let's return it to the script caller.